lib/src/base-styles/_nx-colors.scss

```diff
@@ -59,6 +59,10 @@
   --nx-color-subsection-border: var(--nx-color-border);
   --nx-color-text-dark: var(--nx-color-text-stark);
 
+  // NxGlobalHeader
+  --nx-color-global-header-background: var(--nx-swatch-white);
+  --nx-color-global-header-border: var(--nx-color-border);
```

Comment:
Quick question- in these scenarios, is it better to use the basic swatch color, or a variable name like `--nx-color-border`? I had this thought in the tooltip ticket but kept it as a swatch color. Just to know for the future!

Reply:
> Quick question- in these scenarios, is it better to use the basic swatch color, or a variable name like `--nx-color-border`? I had this thought in the tooltip ticket but kept it as a swatch color. Just to know for the future!

Yeah, it's tricky. My thought was that maybe we wanted to stick with its semantic associations(variables) as much as possible.
And in this case, it was already associated with `--nx-color-border` before, so I kept it.
I think we use swatches if there is no new or previous semantic or general associations

Reply:
thanks!